sites/main-site/src/content/blog/2025/tools-3_4_0.mdx

```diff
@@ -0,0 +1,215 @@
+
+# Pipeline template changes
+
+This release includes also some changes to the pipeline template.
+
+### CI
+
+- We switched the `download_pipeline.yml` action to use the latest version of nf-core/tools instead of `dev`.
+- We fixed the failing `aws{full}test.yml` action by switching to organization-wide variables as inputs.
+
+### Release announcements
+
+Mastodon announcements now include the pipeline description, providing more context when pipelines are released.
+
+# Miscellaneous
+
+## Modules linting
+
+Modules can now use Nextflow's `exec:` blocks and we lint for possible conflicts with `shell:` blocks.
+
+Thanks to [@muffato](https://github.com/muffato) for implementing this feature!
+
+## Linting
+
+The linter now uses the organization specified in `.nf-core.yml` when checking manifest names, homepages, and MultiQC config comments.
+This makes it easier to fork and customize nf-core pipelines for your own organization while maintaining proper attribution.
+
+Thank you [@rrahn](https://github.com/rrahn) for implementing this feature!
+
+## Updated dependencies
+
+- **python**: 3.9 reached end of life, so we set 3.10 as the minimum version and also expanded to support Python 3.14
+- **nf-schema**: bumped to 2.5.0 with improved help message creation for future Nextflow versions
+- **Minimum Nextflow version bumped to 25.04.0** to support the latest features
+- **nf-test**: updated to 0.9.3 with the latest bug fixes and improvements
+- **MultiQC**: updated to 1.31 with new modules and features
+
+# Changelog
+
+You can find the complete changelog and technical details [on GitHub](https://github.com/nf-core/tools/releases/tag/3.4.0).
+
+# Getting help to update your pipeline template
+
+All the template changes are listed in [the Template section of the changelog](https://github.com/nf-core/tools/releases/tag/3.4.0).
+Below, we have collected the most common merge conflicts that you may find and how to resolve them.
+
+## `conf/arm.config` → `conf/emulate_amd64.config`
+
+The generic `arm` profile has been replaced with a more specific `emulate_amd64` profile for better ARM64 architecture handling.
+
+#### Resolution
+
+If you had custom changes in your `arm` profile, maintain them when migrating to the new `emulate_amd64` profile configuration.
+
+## Test profiles
+
+#### Resolution
+
+Keep all your custom changes to test profiles while accepting updates from the template.
+
+## `nextflow.config`
+
+We have updated several key dependencies and configurations:
+
+- Minimum Nextflow version bumped to 25.04.0
+- nf-schema updated to 2.5.0
+- MultiQC updated to 1.31
+
+#### Resolution
+
+Always accept changes from the template for `nextflow.config` as a baseline, then re-apply your custom modifications.
+Make sure to keep any additional plugins you previously added to the configuration.
+
+## `.nf-core.yml`
+
+There might be conflicts in the `repository_type: pipeline` or `template:` sections.
```

Comment:
We didn't change the `.nf-core.yml` with this release 🤔 but ok to keep this section if you saw conflicts, it might be due to the sync command